src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/UpdateOrgContactInformationIntegrationTest.java

```diff
@@ -0,0 +1,284 @@
+
+    @Test
+    void shouldReturn400WhenContactInformationRequestIsNull() {
+        String organisationIdentifier = createOrganisationRequest();
+        Map<String, Object> updateResponse = professionalReferenceDataClient
+            .updateOrgContactInformation(null,  hmctsAdmin,organisationIdentifier,
+                true,true,"");
+
+        assertThat(updateResponse).containsEntry("http_status", "400");
+        deleteOrganisation(organisationIdentifier);
+    }
+
+    @Test
+    void shouldReturn400InvalidDatainRequest() {
+        String organisationIdentifier = createOrganisationRequest();
+        ContactInformationCreationRequest contactInformationCreationRequest =
+            createContactInformationRequest().addressLine1(null).build();
+
+        Map<String, Object> updateResponse = professionalReferenceDataClient
+            .updateOrgContactInformation(contactInformationCreationRequest,
+                hmctsAdmin,organisationIdentifier,true,true,"");
+
+        assertThat(updateResponse).containsEntry("http_status", "400");
+        assertThat(updateResponse.get("response_body").toString()).contains(
+            "Field error in object 'contactInformationCreationRequest' on field 'addressLine1'");
+        deleteOrganisation(organisationIdentifier);
+    }
+
+    @Test
+    void shouldReturn200WhenUpdateMultipleAddressesWithoutDxAddress() {
+
+        List<ContactInformationCreationRequest> contactInformationCreationRequest =
+            getContactInformationListWithoutDxAddress();
+
+        java.util.Map<String, Object> responseForOrganisationCreation = professionalReferenceDataClient
+            .createOrganisation(organisationRequestWithMultipleAddressAllFields().build());
+        String orgIdentifier = (String)responseForOrganisationCreation.get(ORG_IDENTIFIER);
+        java.util.Map<String, Object> retrieveOrganisationResponse = professionalReferenceDataClient
+            .retrieveSingleOrganisation(orgIdentifier, hmctsAdmin);
+
+        List existingContacts = (List)retrieveOrganisationResponse.get("contactInformation");
+        LinkedHashMap existing = (LinkedHashMap)existingContacts.get(0);
+        String addressId = existing.get("addressId").toString();
+
+        Map<String, Object> updateResponse = professionalReferenceDataClient
+            .updateOrgContactInformation(contactInformationCreationRequest.get(0),
+                hmctsAdmin,orgIdentifier,false,true,
+                addressId);
+
+        java.util.Map<String, Object> retrieveOrganisationResponseUpdated = professionalReferenceDataClient
+            .retrieveSingleOrganisation(orgIdentifier, hmctsAdmin);
+
+        List existingContactsUpdated = (List)retrieveOrganisationResponseUpdated.get("contactInformation");
+        LinkedHashMap existingUpdated = (LinkedHashMap)existingContactsUpdated.get(0);
+
+        assertThat(updateResponse).containsEntry("http_status", 200);
+
+        assertThat(existingUpdated.get("addressLine1").toString())
```

Comment:
Old and new data are same, add prefix "updated" to each new field so that it is easy to identify what is updated data.